src/Logger.php

```diff
@@ -19,9 +19,165 @@ class Logger extends \Monolog\Logger
+	}
+
+	/**
+	 * Adds a log record at the ERROR level.
+	 *
+	 * This method allows for compatibility with common interfaces.
+	 *
+	 * @param string $message The log message
+	 * @param array  $context The log context
+	 */
+	public function addError($message, array $context = []): void
+	{
+		$this->error($message, array_merge(['channel' => $this->name], $context));
+	}
+
+	/**
+	 * Adds a log record at the CRITICAL level.
+	 *
+	 * This method allows for compatibility with common interfaces.
+	 *
+	 * @param string $message The log message
+	 * @param array  $context The log context
+	 */
+	public function addCritical($message, array $context = []): void
+	{
+		$this->critical($message, array_merge(['channel' => $this->name], $context));
+	}
+
+	/**
+	 * Adds a log record at the ALERT level.
+	 *
+	 * This method allows for compatibility with common interfaces.
+	 *
+	 * @param string $message The log message
+	 * @param array  $context The log context
+	 */
+	public function addAlert($message, array $context = []): void
+	{
+		$this->alert($message, array_merge(['channel' => $this->name], $context));
+	}
+
+	/**
+	 * Adds a log record at the EMERGENCY level.
+	 *
+	 * This method allows for compatibility with common interfaces.
+	 *
+	 * @param string $message The log message
+	 * @param array  $context The log context
+	 */
+	public function addEmergency($message, array $context = []): void
+	{
+		$this->emergency($message, array_merge(['channel' => $this->name], $context));
+	}
+
+	/**
+	 * Adds a log record at the WARNING level.
+	 *
+	 * This method allows for compatibility with common interfaces.
+	 *
+	 * @param string $message The log message
+	 * @param array  $context The log context
+	 */
+	public function warn($message, array $context = []): void
+	{
+		$this->warning($message, array_merge(['channel' => $this->name], $context));
+	}
+
+	/**
+	 * Adds a log record at the ERROR level.
+	 *
+	 * This method allows for compatibility with common interfaces.
+	 *
+	 * @param string $message The log message
+	 * @param array  $context The log context
+	 */
+	public function err($message, array $context = []): void
+	{
+		$this->error($message, array_merge(['channel' => $this->name], $context));
+	}
+
+	/**
+	 * Adds a log record at the CRITICAL level.
+	 *
+	 * This method allows for compatibility with common interfaces.
+	 *
+	 * @param string $message The log message
+	 * @param array  $context The log context
+	 */
+	public function crit($message, array $context = []): void
+	{
+		$this->critical($message, array_merge(['channel' => $this->name], $context));
+	}
+
+	/**
+	 * Adds a log record at the EMERGENCY level.
+	 *
+	 * This method allows for compatibility with common interfaces.
+	 *
+	 * @param string $message The log message
+	 * @param array  $context The log context
+	 */
+	public function emerg($message, array $context = []): void
+	{
+		$this->emergency($message, array_merge(['channel' => $this->name], $context));
+	}
```

Comment:
I don't quite understand why you're adding all of these aliases here. Are you sure we need them? (I don't understand this library enough to decide myself.)

Reply:
Neither do I, @JanGalek care to comment?

Reply:
All aliases were dropped by Monolog 2.0 in behalf of PSR interface